build/builtin_actors_gen.go

```diff
@@ -29,9 +30,10 @@ var EmbeddedBuiltinActorsMetadata []*BuiltinActorsMetadata = []*BuiltinActorsMet
 		"verifiedregistry": MustParseCid("bafk2bzacebu4joy25gneu2qv3qfm3ktakzalndjrbhekeqrqk3zhotv6nyy2g"),
 	},
 }, {
-	Network:     "butterflynet",
-	Version:     9,
-	ManifestCid: MustParseCid("bafy2bzaceba5qgs4z3imhlxwds5vamahngatvuuglbv5yl3ftfiosj6ud5chs"),
+	Network:      "butterflynet",
+	Version:      9,
+	BundleGitTag: "dev/20230207-pre-rc.7",
```

Comment:
hrmm for butterfly we likely want to start with the mainnet v9 actor and migrate to this new v10 bundle for testing purpose

Reply:
oh no i think its just the wrong tag - i will create an issue